### ISSUE TITLE
Question regarding RGB Full vs. Ycbcr 444 (limited) on LG wOLEDs displays specifically

### ISSUE THREAD

**Issue by DitherMan420** [Maassoft/ColorControl#269]
Hello,

I was told recently that on LG wOLEDs specifically, Ycbcr444 (which is limited) will look slightly better than RGB Full, due to LG's lack of 100% rgb pass-through/how they process ycbcr (not 100% clear on the reason why, just the end result). I was also told that if you do choose to use rgb, that rgb full will look better than limited.

Do you agree with this? If so, do you know why Ycbcr can look just as good (or slightly better) as rgb full, even though ycbcr is always limited? 

So to summarize:

1. Do you agree that ycbcr limited slightly better than rgb full on LG wOLEDs?
2. If so, how can this be since ycbcr is limited and rgb full is, well, full. 

Thank you!

**Comment by Maassoft** [comment]
Assuming you're using a PC as a source:

> 1. Do you agree that ycbcr limited looks slightly better than rgb full on LG wOLEDs?

Maybe, but this is probably caused by dithering of the GPU that is always applied when using a limited range. The PC internally still uses the full range and dithering is applied before compressing the range from 0-255 to 16-235. This might reduce banding on the TV if the TV could not resolve all 256 values correctly.

> 2. If so, how can this be true, since ycbcr is limited and rgb full is, well, full. Shouldn't "full" always look better than "limited", even if ycbcr?

See previous answer.
